addons/mail/static/src/models/messaging_initializer/messaging_initializer.js

```diff
@@ -287,13 +290,15 @@ function factory(dependencies) {
          * @param {Object} currentGuest
          * @param {Object} current_partner
          * @param {integer} current_user_id
+         * @param {boolean} is_portal_user
          * @param {Object} partner_root
          * @param {Object[]} [public_partners=[]]
          */
         _initPartners({
             currentGuest,
             current_partner,
             current_user_id: currentUserId,
+            is_portal_user,
```

Comment:
But instead of introducing a new field, shouldn't it simply use existing field `isInternalUser`? Because this is what is really meant: only internal users can update description.